spec/menu_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'menu'
+
+describe Menu do
+  it 'will have a menu of dishes and prices that can be read' do
+    expect(subject.menu_items).to be_a(Hash)
```

Comment:
Don't test for explicit types. If you need to change this type later on, this test will fail.

Better is:

```ruby
example_object = # construct example object
expect(test).to eq example_object
```